Swift.sublime-syntax

```diff
@@ -344,10 +344,10 @@ contexts:
       scope: keyword.operator
     - match: \w+
       scope: support.class
-    - match: '>'
+    - match: '>\s*'
       pop: true
   class_type:
-    - match: \w+
+    - match: \w+\s*
```

Comment:
My setting says my theme is
`"color_scheme": "Packages/User/SublimeLinter/Monokai (SL).tmTheme"`, it doesn't seem to matter if I removed the linter package it still points to that folder.

Scope names:
![image](https://cloud.githubusercontent.com/assets/7380931/14631896/043a5840-0659-11e6-90b5-027a94179134.png)